.changeset/neat-deers-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': patch
+---
+
+Fixes favicon url parsing of query and fragment
```

Comment:
Suggesting a small tweak to this changeset:

```suggestion
Fixes support for favicon URLs that contain a search query and/or hash
```

Reply:
Will adjust once other feedback is resolved and final names determined.

Reply:
Updated PR.